client/action/favorites/favoritesAction.js

```diff
@@ -7,8 +7,10 @@ import { setFetching, unsetFetching } from '../fetching';
 /**
  * GET all favorite recipes action creator
  *
- * @param {onject} favorites
+ * @param {Object} favorites
```

Comment:
@fuchodeveloper add a short description for all parameters in your jsdoc. 

For reference: http://usejsdoc.org/tags-param.html